docassemble/MAEvictionMoratorium/data/questions/prose_mtd_non_emergency_merge.yml

```diff
@@ -98,6 +97,20 @@ code: |
 
   users[0].sign_on_device
 ---
+code: |
+  who_proxy_sign_for = [codef for codef in codefendants if codef.sign_method == 'proxy']
+---
+code: |
+  who_else_on_device = [codef for codef in codefendants if codef.sign_method == 'local']
+---
+code: |
+  # Get signers that will sign on this device
+  local_signers = users + who_proxy_sign_for + who_else_on_device
+---
+code: |
+  # Used in here and for sending out for signatures (multiuser code)
+  remote_signers = all_signers.difference(local_signers)
+---
```

Comment:
These can probably be combined into one code block. I can't imagine them being used in isolation - a change to one could mean a change to all of them. Might have a 'set' variable of `set_signers` or something. Might include `all_signers` in here too.

This is the way they are for now, though.